fgcm/fgcmExposureSelector.py

```diff
@@ -36,6 +37,12 @@ def selectGoodExposures(self):
 
         self.fgcmPars.expFlag[:] = 0
 
+        bad, = np.where(self.fgcmPars.expFwhm > self.expFwhmCut[self.fgcmPars.expBandIndex])
+        self.fgcmPars.expFlag[bad] |= expFlagDict['BAD_FWHM']
+        self.fgcmLog.info('Flagged %d bad exposures with bad fwhm.' % (bad.size))
+        if not self.quietMode:
+            logFlaggedExposuresPerBand(self.fgcmLog, self.fgcmPars, 'BAD_FWHM')
```

Comment:
Do you really want to lock into : "too large" == "bad"?  I'm just thinking of the case of "too-good-seeing" which is "bad" but because "too small"...